bc_obps/registration/tests/test_01_models.py

```diff
@@ -19,6 +19,16 @@
     MultipleOperator,
     AppRole,
 )
+from model_bakery import baker
+from localflavor.ca.models import CAPostalCodeField
+from datetime import datetime
+
+
+def mock_postal_code():
```

Comment:
I think in Brianna's latest PR (that's in develop now), she did some refactoring so that we don't have to mock postal codes in this way - think there's a util now that can be used everywhere. Would recommend taking a look at that.